src/migrations/migrations/deleteNFTsReactQueryCache.ts

```diff
@@ -0,0 +1,17 @@
+import { Migration, MigrationName } from '../types';
+import store from '@/redux/store';
+import { queryClient } from '@/react-query';
+import { nftsQueryKey } from '@/resources/nfts';
+
+export function deleteNFTsReactQueryCache(): Migration {
+  return {
+    name: MigrationName.deleteNFTsReactQueryCache,
+    async migrate() {
+      const { wallets } = store.getState().wallets;
+
+      const walletAddresses = wallets ? Object.values(wallets).flatMap(wallet => wallet.addresses.map(account => account.address)) : [];
+
+      walletAddresses.forEach(address => queryClient.resetQueries(nftsQueryKey({ address })));
```

Comment:
nfts react query cache currently contains paginated data, so we need to wipe that since it's not compatible with new logic